app/views/dashboards/pro.html.erb

```diff
@@ -18,8 +18,8 @@
 
   <section class="header-card card">
     <h1 class="pro-header">Pro Dashboard for <%= @user_or_org.name %></h1>
-    <p>Welcome to the Pro Dashboard, the home of in-depth user metrics so that authors can make data-driven decisions about the developer ecosystem.</p>
-    <p>This dashboard highlights deep insights especially relevant to developer relations, authors, and serious bloggers.</p>
+    <p>Welcome to the Pro Dashboard, the home of in-depth user metrics so that authors can make data-driven decisions about the  <%= SiteConfig.community_member_label %> ecosystem.</p>
+    <p>This dashboard highlights deep insights especially relevant to  <%= SiteConfig.community_member_label %> relations, authors, and serious bloggers.</p>
```

Comment:
I was very two minded about whether to also add a helper called `community_member_label` like I did with `community_members_label`

Reply:
It seems a bit weird to be referencing SiteConfig.community_member_label for the singular version and community_members_label for the plural. Any thoughts?

Reply:
You could call `pluralize` directly in-line instead of wrapping it in a method, if that makes it feel less weird?

Reply:
Yeah that's also an option. I thought it was a bit repetitive to keep calling pluralize coz the most common use case is to use it in its plural form

Reply:
Yeah I tend to agree with you. I think it’s okay to wrap in a method. You could also wrap the singular one in a method too, if that feels better? 🤷🏽‍♀️ but what you already have doesn’t feel bad to me :)

Reply:
Thanks for this insight, it helps to hear a different perspective. I think since it feels okay, I'm going to leave it as is for now .

Reply:
hahaha I just added a note about this before reading this conversation